api/clients/v2/payload_disperser.go

```diff
@@ -23,6 +29,63 @@ type PayloadDisperser struct {
 	certVerifier    verification.ICertVerifier
 }
 
+// BuildPayloadDisperser builds a PayloadDisperser from config structs.
+func BuildPayloadDisperser(log logging.Logger, payloadDispCfg PayloadDisperserConfig, 
+	dispClientCfg *DisperserClientConfig,
+	ethCfg *geth.EthClientConfig,
+	kzgConfig *kzg.KzgConfig, encoderCfg *encoding.Config) (*PayloadDisperser, error) {
+	
+	// 1 - verify key semantics and create signer
+	var signer core.BlobRequestSigner
+	if len(payloadDispCfg.SignerPaymentKey) == 64 {
+		signer = auth.NewLocalBlobRequestSigner(payloadDispCfg.SignerPaymentKey)
```

Comment:
Confused why this constructor doesn't return an error if parsing fails... I'm gonna go ahead and open a PR to fix this (though it shouldn't hold up your PR here)